lib/maxmind/geoip2/record/continent.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'maxmind/geoip2/record/place'
+
+module MaxMind::GeoIP2::Record
+  # Contains data for the continent record associated with an IP address.
+  #
+  # This record is returned by all location services and databases.
+  #
+  # See Place for inherited methods.
+  class Continent < Place
+    # Returns a two character continent code like "NA" (North America) or "OC"
```

Comment:
"Returns" is probably unnecessary. Some accessors have it and others just say something like "A two character..."

Reply:
True. I updated several of the comments to be more consistent!